sbi/simulators/mp_simutils.py

```diff
@@ -1,39 +1,46 @@
 import multiprocessing as mp
 import numpy as np
 from tqdm.auto import tqdm
-from typing import Callable
+from typing import Callable, Tuple
 import torch
 from sys import platform
+from torch import Tensor
 
 
 def simulate_mp(
     simulator: Callable,
-    theta: torch.Tensor,
+    theta: Tensor,
     num_workers: int = 4,
     worker_batch_size: int = 20,
     verbose: bool = False,
     show_progressbar: bool = True,
-) -> (torch.Tensor, torch.Tensor):
+) -> Tuple[Tensor, Tensor]:
     """
-    Wrapper function for the callable simulator to allow it to use multiprocessing.
+    Return parameters theta and simulated data x, simulated using multiprocessing on the
+        passed simulator.
 
     Args:
         simulator: simulator function that will be executed on each worker
         theta: all simulation parameters
         num_workers: number of parallel workers to start
-        worker_batch_size: how many thetas are processed on each worker. This number of
-            thetas will then be handled by simulate_in_batches()
+        worker_batch_size: Number of parameters that are processed per worker. A worker
```

Comment:
Can it be confusing? (individual parameters vs. parameter batches/sets/...)

Reply:
How about writing `parameter vectors` instead of `parameters`?